build.gradle

```diff
@@ -78,9 +77,12 @@ dependencies
     shaded project.configurations.getByName('compile')
     shaded packages.slf4j.log4j12
     shaded packages.log4j
+
+    // Support JUnit 3/4 tests
+    compile packages.junit.junit4
```

Comment:
Can this one be with all the other compile up top? Just a nit.

Reply:
Sure.